convene-web/spec/models/demo_workspace_spec.rb

```diff
@@ -2,6 +2,8 @@
 
 RSpec.describe DemoWorkspace, type: :model do
   describe ".prepare" do
+    before { allow(ENV).to receive(:[]).with('DEMO_ENABLED').and_return('true') }
```

Comment:
Oh nice catch. Maybe we want this in it's own context for `When the environment is set to enable demo` and then there could be the inverse with an it.